lib/foreman/logging.rb

```diff
@@ -259,7 +259,15 @@ def indent_lines(string)
 
     class MultilineRequestPatternLayout < MultilinePatternLayout
       def indent_lines(string)
-        string.gsub("\n", "\n #{::Logging.mdc['request'].split('-').first} | ")
+        if request_id
+          string.gsub("\n", "\n #{request_id.split('-').first} | ")
+        else
+          super(string)
+        end
+      end
+
+      def request_id
```

Comment:
Should this be a private method?

Reply:
I don't think it needs to be public, so I can make it private